workshop/content/japanese/30-python/60-cleanups/_index.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "クリーンアップ"
+weight = 60
+chapter = true
++++
+
+# スタックのクリーンアップ
```

Comment:
Please update this cleanup file to be consistent with english version. Like the Typescript cleanup page in Japanese, the section on removal policy is missing 🙂